### PR TITLE
6657

### PR DESCRIPTION
**Pull request by OLGKAD**
Fixes #

**PR Checklist**

Ensure that you have:
- [ ] Read and understood our PR guideline: https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr
  - [ ] Added the issue number to the "Fixes" keyword above
  - [ ] Titled the PR as specified in the abovementioned document
- [ ] Made your changes on a branch other than `master` and `release`
- [ ] Gone through all the changes in this PR and ensured that:
  - [ ] They addressed one (and only one) issue
  - [ ] No unintended changes were made
- [ ] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [ ] Added/updated tests, if changes in functionality were involved
- [ ] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

### CODE REVIEW

File: src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java [hunk 1]
```diff
@@ -248,7 +248,17 @@ private String getInstructorQuestionResultsStatisticsHtml(
                         Slots.SUMMARY_TITLE, statsTitle,
                         Slots.STATS_FRAGMENTS, fragmentHtml.toString());
     }
-
+    ////////////////////////////////////////////////////////////////////
```
Comment (Lucasljungberg) [inline src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java]: Are these lines of comments an accident?

File: src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java [hunk 2]
```diff
@@ -265,8 +275,11 @@ private String getStudentQuestionResultsStatisticsHtml(
         // need to know which recipients are hidden since anonymised recipients will not appear in the summary table
         List<String> hiddenRecipients = getHiddenRecipients(responses, question, bundle);
 
+        System.out.println("Called function");
```
Comment (Lucasljungberg) [inline src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java]: Try to remove some outputs

File: src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java [hunk 3]
```diff
@@ -447,6 +460,17 @@ private String getAverageExcludingSelfText(boolean showAvgExcludingSelf, Decimal
         return "";
     }
 
+////////////////////////////////////////////////////////////////////
```
Comment (Lucasljungberg) [inline src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java]: If these aren't mean to be here, you should remove them too.

File: src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java [hunk 4]
```diff
@@ -528,7 +573,17 @@ private boolean showAverageExcludingSelf(
         }
         return false;
     }
-
+    ////////////////////////////////////////////////////////////////////
```
Comment (Lucasljungberg) [inline src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java]: If these aren't mean to be here, you should remove them too.

File: src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java [hunk 5]
```diff
@@ -537,7 +592,9 @@ private void populateSummaryStatisticsFromResponses(
             Map<String, Integer> numResponses,
             Map<String, Integer> numResponsesExcludingSelf) {
 
+        System.out.println("ALSKDJ");
```
Comment (Lucasljungberg) [inline src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java]: You should remove the outputs (it wasn't you who put them there though, but it's still good to do it ^^ )

File: src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java [hunk 6]
```diff
@@ -599,7 +658,9 @@ private void populateSummaryStatisticsFromResponses(
             FeedbackSessionResultsBundle bundle) {
         List<String> hiddenRecipients = new ArrayList<>(); // List of recipients to hide
         FeedbackParticipantType type = question.recipientType;
+
         for (FeedbackResponseAttributes response : responses) {
+            if (!bundle.visibilityTable.containsKey(response.getId())) continue;
```
Comment (Lucasljungberg) [inline src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java:631]: I think this part is handled by another branch.

**Review by Lucasljungberg — CHANGES_REQUESTED** [review]
Review the comments.

File: src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java (hunk 6 above)
Comment (OLGKAD) [reply src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java:631]: Thanks for the comments! Should be fixed now.

**Review by Lucasljungberg — APPROVED** [review]

**Pull request merged**